source/libs/linkify-text-nodes.ts

```diff
@@ -60,5 +61,9 @@ export default (
 
 			textNode.replaceWith(linkified);
 		}
+
+		if (el instanceof Element) {
```

Comment:
A call to this utility function is made from within `options.ts`. The passed type is `DocumentFragment` but `zipTextNodes` requires `Element`. Sadly, `DocumentFragment` is not of type element. So I added this check. It won't change the behaviour or signatures of other functions that rely on this method.